modules/gitops-argo-cd-notification.adoc

```diff
@@ -2,23 +2,54 @@
 //
 // * argocd_instance/argo-cd-cr-component-properties.adoc
 
-:_content-type: PROCEDURE
+:_mod-docs-content-type: PROCEDURE
 [id="gitops-argo-cd-notification_{context}"]
 = Enabling notifications with Argo CD instance
 
-To enable or disable the link:https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/[Argo CD notifications controller], set a parameter in the Argo CD custom resource. By default, notifications are disabled. To enable notifications, set the `enabled` parameter to `true` in the `.yaml` file:
+Argo CD notifications allow you to send notifications to external services when events occur in your Argo CD instance. For example, you can send notifications to Slack or email when a sync operation fails. By default, notifications are disabled in Argo CD instances.
+
+.Prerequisites
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {OCP} web console.
+* You have installed the {gitops-title} Operator in your cluster.
 
 .Procedure
 
-. Set the `enabled` parameter to `true`:
+To enable notifications for an Argo CD instance using the {OCP} web console, complete the following steps:
 
+. In the *Administator* perspective of the {OCP} web console, go to *Operators* → *Installed Operators*.
+. From the list of *Installed Operators*, select the {gitops-title} Operator, and then click on the *ArgoCD* tab.
+. Select the Argo CD instance name for which you want to enable notifications, for example `openshift-gitops`.
```

Comment:
I think it might be clearer to say
```suggestion
. Select the Argo CD instance name you want to enable notifications for. For example, `openshift-gitops`.
```

Reply:
Addressed.

thank you.